### PR TITLE
Try to fix common workflow and enable parallel for smoketest

### PR DESCRIPTION
**Pull request by isabelrios**
Still Common workflow and RunAllXCUITests are failing on main... Also trying to enable parallel testing on Smoke to reduce time

### CODE REVIEW

**Comment by isabelrios** [comment]
Tests are running now for the RunAll workflow: https://app.bitrise.io/build/7c36277437c79926#?tab=log

File: bitrise.yml [hunk 1]
```diff
@@ -2929,7 +2923,7 @@ trigger_map:
 - push_branch: v31.x
   workflow: xcode12-release-and-beta-nocache
 - pull_request_target_branch: main
-  workflow: RunUnitTests
+  workflow: RunAllXCUITests
```
Comment (isabelrios) [inline bitrise.yml:2926]: This has to go back to its original value before merging...

File: XCUITests/SmokeXCUITests.xctestplan [hunk 2]
```diff
@@ -13,6 +13,7 @@
   },
   "testTargets" : [
     {
+      "parallelizable" : true,
```
Comment (isabelrios) [inline XCUITests/SmokeXCUITests.xctestplan:16]: Not sure if we want to enable this... test execution will be reduced around ~7min but I'm afraid there are going to be some intermittents due to the parallel testing as [in this build ](https://app.bitrise.io/build/7cf5d010cee30d2e#?tab=log)and also locally... wdyt @AaronMT?

Comment (AaronMT) [reply XCUITests/SmokeXCUITests.xctestplan:16]: I don't know anything about this.

**Review by AaronMT — APPROVED** [review]
WFM. I don't know anything about parallelizable though.

File: XCUITests/SmokeXCUITests.xctestplan (hunk 2 above)
Comment (isabelrios) [reply XCUITests/SmokeXCUITests.xctestplan]: It usually happens that some tests are flaky when running this way... if we enable this we should be more agressive with disabling/fixing those tests... Maybe not good timing for this.. we can always try...

**Review by AaronMT — APPROVED** [review]
If a push works with that user we can run the CURL/jq step manually to verify output and make any changes if need be.

**Pull request merged**